node/tests/RedisClusterClient.test.ts

```diff
@@ -274,7 +274,7 @@ describe("RedisClusterClient", () => {
         TIMEOUT,
     );
 
-    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+    it.only.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
```

Comment:
AFAIK this will make all other tests ignored

Reply:
Obviously, it wasn't deleted by mistake.